src/wp-includes/script-loader.php

```diff
@@ -1051,6 +1060,8 @@ function wp_default_scripts( $scripts ) {
 
 	$scripts->add( 'admin-bar', "/wp-includes/js/admin-bar$suffix.js", array( 'hoverintent-js' ), false, 1 );
 
+	$scripts->add( 'go-back', "/wp-includes/js/go-back$suffix.js", array(), false, 1 );
```

Comment:
I'm a bit skeptical about adding an extra script file just for this one functionality.

It's only used in `setup-config` and the `wp_die` handlers.

For the former, having it named `setup-config` seemed fine.

For `wp_die`, I don't think we can even enqueue the script. `wp_die()` could be used before `wp_print_scripts` is even defined. I would just inline the script there if possible.

Not sure if `wp_print_script_loader_tag` would be available in `wp_die` either.

Reply:
You're right. 
I think `wp_print_script_loader_tag` would be available as `wp_die` is defined in the same file (functions.php).
But I have a limited understanding of the WP architecture, and I may be wrong.

Reply:
Oh, then it would work.